### PR TITLE
Remove stale coco artifacts

### PR DESCRIPTION
**Pull request by UnmeshDeodhar**
Remove coco artifacts.

Fixes: #5502

Signed-of-by: Unmesh Deodhar <udeodhar@amd.com>

### CODE REVIEW

**Comment by fidencio** [comment]
This works well, but I think we should also remove some links we create during the installation.
For instance, if you go to the install scripts, you'll notice we end up linking to binaries / ovmf / and whatnot to /usr/local/bin.  Please, take a look at those scripts and also clean up the scripts we create there.

**Comment by UnmeshDeodhar** [comment]
@fidencio I have added the changes that delete and unlink binaries from /opt/confidential-containers please let me know if this looks okay.

File: .ci/lib.sh [hunk 1]
```diff
@@ -464,6 +464,23 @@ gen_clean_arch() {
 	# make sure cargo and rustup home dir are removed
 	info "remove cargo and rustup home dir"
 	sudo rm -rf ~/.cargo ~/.rustup
+
+	# make sure coco artifacts are removed
+	info "remove coco artifacts"
+	sudo rm -rf /opt/confidential-containers
```
Comment (fidencio) [inline .ci/lib.sh:470]: So, here you removed everything from `/opt/confidential-containers` ...

File: .ci/lib.sh [hunk 2]
```diff
@@ -464,6 +464,23 @@ gen_clean_arch() {
...
+
+	# make sure to clean up binaries
+	unlink /opt/confidential-containers/share/ovmf
```
Comment (fidencio) [inline .ci/lib.sh:473]: ... and then here you're trying to unlink a dir that was just removed in a few lines above.

As this pattern repeats in all the removals, so I'd suggest to move the `sudo rm -rf /opt/confidential-containers` to be the last part of the cleanup.

File: .ci/lib.sh [hunk 3]
```diff
@@ -464,6 +464,23 @@ gen_clean_arch() {
...
+	sudo rm -f /usr/share/ovmf
+
+	unlink /opt/confidential-containers/share/td-shim
+	sudo rm -f /usr/share/td-shim
+
+	unlink /opt/confidential-containers/share/tdvf
+	sudo rm -f /usr/share/tdvf
+
+	unlink /opt/confidential-containers/bin/qemu-system-x86_64
+	sudo rm -f /usr/bin/qemu-system-x86_64
```
Comment (fidencio) [inline .ci/lib.sh:483]: You're missing:
*  `./opt/confidential-containers/bin/qemu-system-x86_64-tdx` which seems to be linked as `/usr/bin/qemu-system-x86_64-tdx`
* `/opt/kata/libexec/virtiofsd`, which is linked as `/usr/libexec/virtiofsd`
`

**Review by fidencio — CHANGES_REQUESTED** [review]
I've left a few comments.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @UnmeshDeodhar!

**Comment by fidencio** [comment]
Cloud Hypervisor TDX is green but didn't report back: http://10.112.240.208:8080/job/tests-CCv0-cloud-hypervisor-TDX-CRI_CONTAINERD_K8S-PR/374/console

**Pull request merged**